ui/modules/mon-espace/SIFA/SIFAPage.tsx

```diff
@@ -115,17 +97,33 @@ const SIFAPage = ({ isMine }) => {
   }
 
   return (
-    <Flex flexDir="column" width="100%" my={10}>
+    <Container maxW="xl" p="8">
       <Flex as="nav" align="center" justify="space-between" wrap="wrap" w="100%" alignItems="flex-start">
-        <Heading textStyle="h2" color="grey.800" mt={5} mb={8}>
-          {isMine ? "Mon Enquête SIFA" : "Son Enquête SIFA"}
+        <Heading as="h1" color="#465F9D" fontSize="beta" fontWeight="700" mb={8}>
+          {props.modePublique ? "Son" : "Mon"} Enquête SIFA
         </Heading>
         <HStack spacing={4}>
           <DownloadButton
-            fileName={exportSifaFilename}
-            getFile={() => _getBlob(`/api/v1/organismes/${organisme._id}/sifa-export`)}
-            title="Télécharger SIFA"
-          />
+            variant="secondary"
+            action={async () => {
+              trackPlausibleEvent("telechargement_sifa");
+              downloadObject(
+                await _getBlob(`/api/v1/organismes/${organisme._id}/sifa-export`),
+                `tdb-données-sifa-${
+                  organisme.enseigne ?? organisme.raison_sociale ?? "Organisme inconnu"
+                }-${new Date().toLocaleDateString()}.csv`,
+                "text/plain"
+              );
+              const nbEffectifsInvalides = organismesEffectifs.filter(
+                (effectif) => effectif.requiredSifa.length > 0
+              ).length;
+              toastWarning(
+                `Parmi les ${organismesEffectifs.length} effectifs que vous avez déclarés, ${nbEffectifsInvalides} d'entre eux ne comportent pas l'ensemble des informations requises pour l'enquête SIFA. Si vous ne les corrigez/complétez pas, votre fichier risque d'être rejeté. Vous pouvez soit les éditer directement sur la plateforme soit modifier votre fichier sur votre ordinateur.`
+              );
```

Comment:
J'ai pas eu le temps de lire en 5 secondes et je lis vite. Suggestion : 20 secondes j'ai le temps de lire et comprendre le message (surtout la première fois). Et une coche pour fermer si vraiment ça me soule.

```suggestion
              toastWarning(
                `Parmi les ${organismesEffectifs.length} effectifs que vous avez déclarés, ${nbEffectifsInvalides} d'entre eux ne comportent pas l'ensemble des informations requises pour l'enquête SIFA. Si vous ne les corrigez/complétez pas, votre fichier risque d'être rejeté. Vous pouvez soit les éditer directement sur la plateforme soit modifier votre fichier sur votre ordinateur.`,
                {
                  isClosable: true,
                  duration: 20000,
                }
              );
```